### PR TITLE
Fix tqdm encoding error on python2.6

### PR DESCRIPTION
**Pull request by Xuanwo**
Signed-off-by: Xuanwo <xuanwo@yunify.com>

### CODE REVIEW

File: qingstor/qsctl/constants.py [hunk 1]
```diff
@@ -29,3 +31,9 @@
 
 # tqdm bar format
 BAR_FORMAT = "{l_bar}{bar}| {n_fmt}/{total_fmt} [{remaining} ETA, {rate_fmt}]"
+
+# In python2.6, tqdm will raise encoding error, so use ascii instead
+USE_ASCII = is_python26
+
+# Units used in output
+UNITS = ('KiB', 'MiB', 'GiB', 'TiB', 'PiB', 'EiB')
```
Comment (pengsrc) [inline qingstor/qsctl/constants.py]: Why not `UNITS = ('KB', 'MB', 'GB', 'TB', 'PB', 'EB')`

Comment (Xuanwo) [inline qingstor/qsctl/constants.py]: `'KB', 'MB', 'GB', 'TB', 'PB', 'EB'` use 1000 as carry, and `'KiB', 'MiB', 'GiB', 'TiB', 'PiB', 'EiB'` use 1024, which is more correct.

Ref: https://en.wikipedia.org/wiki/Mebibyte

Comment (pengsrc) [inline qingstor/qsctl/constants.py]: Right, why not `1000`

Comment (pengsrc) [inline qingstor/qsctl/constants.py]: OK

**Comment by pengsrc** [comment]
Add test for `py26` would be better.

**Comment by Xuanwo** [comment]
LGTM, I will add it.

File: qingstor/qsctl/constants.py (hunk 1 above)
Comment (pengsrc) [inline qingstor/qsctl/constants.py]: All right, it's a little rarely to see `KiB`.

**Pull request merged**